### PR TITLE
Accept >=1 versus >=5 returns in chaining if else

### LINKED ISSUE #7750: Waypoint: Chaining If Else Statements :should not enforce 5 return statements
Challenge [Chaining If Else Statements](https://www.freecodecamp.com/challenges/chaining-if-else-statements#?solution=function%20myTest(num)%20%7B%0A%20%20%2F%2F%20Only%20change%20code%20below%20this%20line%0A%20%20var%20result%20%3D%20%22%22%3B%0A%20%20if%20(num%20%3C%205)%20%7B%0A%20%20%20%20%20%20result%20%3D%20%22Tiny%22%3B%0A%20%20%7D%20else%20if%20(%20num%20%3C%2010)%20%7B%0A%20%20%20%20result%20%3D%20%22Small%22%3B%0A%20%20%7D%20else%20if%20(num%20%3C%2015)%20%7B%0A%20%20%20%20result%20%3D%20%22Medium%22%3B%0A%20%20%7D%20else%20if%20(num%20%3C%2020)%20%7B%0A%20%20%20%20result%20%3D%20%22Large%22%3B%0A%20%20%7D%20else%20%7B%0A%20%20%20%20result%20%3D%20%22Huge%22%3B%0A%20%20%7D%0A%20%20%0A%20%20return%20result%3B%0A%20%20%2F%2F%20Only%20change%20code%20above%20this%20line%0A%7D%0A%0A%2F%2F%20Change%20this%20value%20to%20test%0AmyTest(7)%3B%0A) has an issue.

For this challenge, a solution with only one return statement should be accepted. It's a good practice to have only one exit point and we shouldn't discourage this solution.

User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64; rv:45.0) Gecko/20100101 Firefox/45.0</code>.

My code:
```javascript
function myTest(num) {
  // Only change code below this line
  var result = "";
  if (num < 5) {
      result = "Tiny";
  } else if ( num < 10) {
    result = "Small";
  } else if (num < 15) {
    result = "Medium";
  } else if (num < 20) {
    result = "Large";
  } else {
    result = "Huge";
  }
  
  return result;
  // Only change code above this line
}

// Change this value to test
myTest(7);

```

### PR DESCRIPTION
**Pull request by erictleung**
Tested locally. Passes `npm run test-challenges`. Closes #7750.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 1]
```diff
@@ -3345,7 +3345,7 @@
       "tests": [
         "assert(code.match(/else/g).length > 3, 'message: You should have at least four <code>else</code> statements');",
         "assert(code.match(/if/g).length > 3, 'message: You should have at least four <code>if</code> statements');",
-        "assert(code.match(/return/g).length >= 5, 'message: You should have at least five <code>return</code> statements');",
+        "assert(code.match(/return/g).length >= 1, 'message: You should have at least five <code>return</code> statements');",
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: `You should have at least five <code>return</code> statements` <-- Change the no of return in the message

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 2]
```diff
@@ -3345,7 +3345,7 @@
       "tests": [
         "assert(code.match(/else/g).length > 3, 'message: You should have at least four <code>else</code> statements');",
         "assert(code.match(/if/g).length > 3, 'message: You should have at least four <code>if</code> statements');",
-        "assert(code.match(/return/g).length >= 5, 'message: You should have at least five <code>return</code> statements');",
+        "assert(code.match(/return/g).length >= 1, 'message: You should have at least one <code>return</code> statements');",
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: @erictleung You are going to probably kill me for this but I have one more request:
`statements` > `statement`

Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: @raisedadead No worries! I'm usually a stickler for grammar. Seems I've gotten careless...

**Comment by raisedadead** [comment]
:sparkles:  AWESOME :sparkles:

**Pull request merged**